pyproject.toml

```diff
@@ -49,6 +49,7 @@ dependencies = [
     "scikit-learn>=0.24.0,<1.2.0",
     "statsmodels>=0.12.1",
     "scipy<1.9.0",
+    "backoff",
```

Comment:
please remove backoff from the core dependencies

Reply:
backoff removed